compiler/src/iree/compiler/GlobalOptimization/Passes.h

```diff
@@ -30,89 +30,86 @@ struct TransformOptions : public PassPipelineOptions<TransformOptions> {
 
-// Create a pass to detach elementwise ops from named Linalg ops.
+/// Demotes inputs (LHS, RHS) of linalg matmul-like ops from f32 to bf16.
+std::unique_ptr<Pass> createDemoteContractionInputsToBF16Pass();
+
+/// Detaches elementwise ops from named Linalg ops.
 std::unique_ptr<Pass> createDetachElementwiseFromNamedOpsPass();
 
-// Apply patterns to erase unused linalg operands and remove dead code
-// associated.
+/// Applies patterns to erase unused linalg operands and remove dead code
+/// associated.
 std::unique_ptr<OperationPass<mlir::ModuleOp>>
 createEraseUnusedLinalgOperands();
 
-// Expands tensor shape dimensions into SSA values across the program.
+/// Expands tensor shape dimensions into SSA values across the program.
 std::unique_ptr<OperationPass<mlir::ModuleOp>> createExpandTensorShapesPass();
 
-// A pass to fuse dequantization and matmul linalg.generic ops
+/// Fuses dequantization and matmul linalg.generic ops
 std::unique_ptr<InterfacePass<mlir::FunctionOpInterface>>
 createFuseDequantizationMatmulPass(
     bool enableQuantizedMatmulReassociation = false);
 
-// A pass to fuse two matmul ops and a linalg.generic Silu op
+/// Fuses two matmul ops and a linalg.generic Silu op
 std::unique_ptr<InterfacePass<mlir::FunctionOpInterface>>
 createFuseSiluHorizontalMatmulPass();
 
-// Create a pass that generalizes some named Linalg ops into `linalg.generic`
-// operations since the IREE compiler can handle that better.
+/// Generalizes some named Linalg ops into `linalg.generic` operations since the
+/// IREE compiler can handle that better.
 std::unique_ptr<InterfacePass<mlir::FunctionOpInterface>>
 createGeneralizeLinalgNamedOpsPass();
 
-// Infers and inserts util.numeric.optional_narrow ops at points that may be
-// beneficial.
+/// Infers and inserts util.numeric.optional_narrow ops at points that may be
+/// beneficial.
 std::unique_ptr<Pass> createInferNumericNarrowingPass();
 
-// Materializes logical encodings to physical encodings if there is a single
-// device target.
+/// Materializes logical encodings to physical encodings if there is a single
+/// device target.
 std::unique_ptr<OperationPass<mlir::ModuleOp>>
 createMaterializeHomogeneousEncodingsPass();
 
-// Optimizes numerics given annotations added via
-// iree-global-opt-infer-numeric-narrowing.
+/// Optimizes numerics given annotations added via
+/// iree-global-opt-infer-numeric-narrowing.
 std::unique_ptr<Pass> createOptimizeNumericsPass();
 
-// Create a pass that propagates linalg.transpose ops to a restricted set of
-// operations.
+/// Propagates linalg.transpose ops to a restricted set of operations.
 std::unique_ptr<InterfacePass<mlir::FunctionOpInterface>>
 createPropagateLinalgTransposePass(bool enableAggressivePropagation = false);
 
-// Create a pass to raise sequence of ops to higher level linalg.ext
-// representation.
+/// Raises sequence of ops to higher level linalg.ext representation.
```

Comment:
[optional] This comment is a bit out of date. Maybe
```
Performs specialized raisings of various sequences of ops to a representation
easier for the compiler to handle.
```

Reply:
SG, thank you!